### ISSUE TITLE
Licenses of all bundled PyPI+NPM dependencies as overview

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri#2321]
## Summary

This is something we should look at, but IMO it doesn't have to be now. We have no conflicts of licenses in our dependencies AFAIK, and there are no one immediately requesting an overview.

But it would be nice with an actual overview. The issue for instance arises in the debian package where I'd like to at least specify a list of relevant licenses.

It could be auto-generated and put in the docs by using the stuff that fetches PyPI license info:

https://github.com/learningequality/kolibri/blob/develop/test/conditional/test_licenses.sh

@indirectlylit were you also working on something for NPM?

**Comment by indirectlylit** [comment]
> were you also working on something for NPM?

we use `yarn` which now has an audit tool. We recently did a manual check and everything looked good

**Comment by indirectlylit** [comment]
related: #1124

@benjaoming what is the concrete outcome of this issue? It sound like you're asking for a command like

```
$ kolibri manage list-dependencies
NPM:
[ {name, version, license}, ... ]
PyPi:
[ {name, version, license}, ... ]
Others:
[ {name, version, license}, ... ]
```

Is that about right?

**Comment by indirectlylit** [comment]
see also https://github.com/learningequality/kolibri/blob/develop/test/conditional/test_licenses.sh

**Comment by benjaoming** [comment]
@indirectlylit I wrote that script, it's already in the issue description :)

I'm thinking that this is a build/test related logic that should be shipped as a management script? If you can add a call to the yarn auditing tool here, I think that's sufficient?

https://github.com/learningequality/kolibri/blob/develop/test/if.sh#L143

**Comment by rtibbles** [comment]
I think the concrete fix here is to add `yarn licenses list` into our `test_licenses.sh`.

**Comment by rtibbles** [comment]
Fixed in #8355
